foundations/installations/installation_overview.md

```diff
@@ -45,7 +45,7 @@ Windows, by itself, **is not natively supported** by The Odin Project, or on our
 
 A **virtual machine** is an emulation of a computer that runs within your existing OS. It allows you to use another operating system inside of a program on your current operating system (e.g. Running Linux inside of Windows). Virtual machines are as straightforward to install as any other program and are risk free. If you don't like Linux, you can easily remove the virtual machine. Virtual machines are a great way for new developers to get started quickly.
```

Comment:
```suggestion
A [virtual machine](https://youtu.be/yIVXjl4SwVo) is an emulation of a computer that runs within your existing OS. It allows you to use another operating system inside of a program on your current operating system (e.g. Running Linux inside of Windows). Virtual machines are as straightforward to install as any other program and are risk free. If you don't like Linux, you can easily remove the virtual machine. Virtual machines are a great way for new developers to get started quickly.
```
Putting the link from line 48 here (and then deleting line 48 as it's no longer needed) will resolve the linting error **and** keep the link text sufficiently descriptive.